index.html

```diff
@@ -0,0 +1,1342 @@
+                    <li><a href="#" class="trending-news-li">JOE BIDEN</a></li>
+                    <li><a href="#" class="trending-news-li">DONALD TRUMP</a></li>
+                    <li><a href="#" class="trending-news-li">INUGURATIONS</a></li>
+                    <li><a href="#" class="trending-news-li">CORONAVIRUS</a></li>
+                    <li><a href="#" class="trending-news-li">STIMULUS PACKAGE</a></li>
+                    <li><a href="#" class="trending-news-li">VACINE</a></li>
+                    <li><a href="#" class="trending-news-li">CAPITOL RIOTS</a></li>
+                </ul>
+            </div>
+        </nav>
+         
+    </header>
+
+    <div class="main row mx-1">
+        <section class="feature-stories col-lg-3 col-md-5 ">
+            <div class='my-story d-md-none'>
+                <h1 class="text-secondary my-1">TOP STORY</h1>
+                <div class="card border-top-0 border-end-0 border-start-0">
+                    <div class="position-relative">
+                        <img class="card-img-top " src="./assets/images/joe-biden-kamala-harris-their-spouses.jpg"
+                            alt="">
+                        <h5 class="card-title cat-header position-absolute bottom-0 start-0 py-1 px-2 bg-light m-0 text-danger fw-bolder">
+                            Politics</h5>
+                    </div>
+                    <div class="card-body p-0 my-2">
+                        <h3 class="card-header bg-white px-1 border-bottom-0 story-h3 fw-bolder ">Joe Biden's First
+                            Executive Actions Undo
+                            Many of Donald Trump's</h3>
+                        <p class="card-text story-p">
+                            Actions related to the COVID-19 crisis, climate change and racial equality are to be among
+                            the first taken as he begins his tenure.
+                        </p>
+                    </div>
+                    <div class="story border-top py-2 d-flex gap-2">
+                        <img class=" story-img " src="./assets/images/us-president-donald-trump.jpg" alt="">
+                        <div class="content">
+                            <h4 class="fw-bold ">Chinese Billionaire Jack Ma Re-emerges
+                            </h4>
+                            <p>China's rock star entrepreneur, who co-founded e-commerce giant Alibaba Group
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <h3 class="feature-h3 text-secondary">FEATURE STORIES</h3>
+            <div class="card border-0 my-3">
+                <div class="position-relative">
+                    <img class="card-img-top " src="./assets/images/trump-farewell.jpg" alt="">
+                    <h5
+                        class="card-title cat-header position-absolute bottom-0 start-0 py-1 px-2  bg-light m-0 text-danger fw-bolder">
```

Comment:
- [ ] I see that you are using the `text-danger` and `button-danger` sometimes to get a red text and background in your clone, while using Bootstrap classes it's great it can be a problem in this case.
The shade of red the NewsWeek site uses is not the same red the Bootstrap classes have.

Please go through your clone and make sure that all the places where the shade of red differ from the original site are fixed.